style-plugin.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const postcss = require('postcss');
+const postCssModules = require('postcss-modules');
+const sass = require('sass');
+
+const CreateStylePlugin = isProd => {
+    return {
+        name: 'style-plugin',
+
+        setup(build) {
+            build.onResolve(
+                {
+                    filter: /^css-module:/,
+                },
+                args => {
+                    const prefixLen = 'css-module:'.length;
+                    const suffixLen = '.scss'.length;
```

Comment:
I can change it to be more generic for sure but I think it would not be possible to get here without either `.scss` or `.css` in the import.

Let me know if you had something else in mind.